ansible/roles/controller/tasks/deploy.yml

```diff
@@ -82,6 +82,7 @@
     - "{{ db.whisk.actions }}"
     - "{{ db.whisk.auth }}"
     - "{{ db.whisk.activations }}"
+  run_once: true
```

Comment:
?

Reply:
The intent of this check is to verify, that the databases exist.
If you have a system with hundred of invokers, I think it's not necessary that each invoker checks, if the database exists. It should be enough, if one invoker does this check.
What do you think?

Reply:
Should we take this out of this PR and open a new one for it? Doesn't seem to have anything to do with the actual change?